diff --git a/Modules/_sqlite/cache.h b/Modules/_sqlite/cache.h
index a133903..71e7629 100644
--- a/Modules/_sqlite/cache.h
+++ b/Modules/_sqlite/cache.h
@@ -25,6 +25,10 @@
 #define PYSQLITE_CACHE_H
 #include "Python.h"

+#ifndef MODULE_NAME
+#define MODULE_NAME "_sqlite3"
+#endif
+
 /* The LRU cache is implemented as a combination of a doubly-linked with a
  * dictionary. The list items are of type 'Node' and the dictionary has the
  * nodes as values. */
diff --git a/Modules/_sqlite/microprotocols.h b/Modules/_sqlite/microprotocols.h
index 99ff6f6..6f45134 100644
--- a/Modules/_sqlite/microprotocols.h
+++ b/Modules/_sqlite/microprotocols.h
@@ -28,6 +28,10 @@

 #include <Python.h>

+#ifndef MODULE_NAME
+#define MODULE_NAME "_sqlite3"
+#endif
+
 /** the names of the three mandatory methods **/

 #define MICROPROTOCOLS_GETQUOTED_NAME "getquoted"
diff --git a/Modules/_sqlite/module.h b/Modules/_sqlite/module.h
index 6f90934..79ce261 100644
--- a/Modules/_sqlite/module.h
+++ b/Modules/_sqlite/module.h
@@ -25,6 +25,10 @@
 #define PYSQLITE_MODULE_H
 #include "Python.h"

+#ifndef MODULE_NAME
+#define MODULE_NAME "_sqlite3"
+#endif
+
 #define PYSQLITE_VERSION "2.6.0"

 extern PyObject* pysqlite_Error;
diff --git a/Modules/_sqlite/prepare_protocol.h b/Modules/_sqlite/prepare_protocol.h
index 924e162..e666fa1 100644
--- a/Modules/_sqlite/prepare_protocol.h
+++ b/Modules/_sqlite/prepare_protocol.h
@@ -25,6 +25,10 @@
 #define PYSQLITE_PREPARE_PROTOCOL_H
 #include "Python.h"

+#ifndef MODULE_NAME
+#define MODULE_NAME "_sqlite3"
+#endif
+
 typedef struct
 {
     PyObject_HEAD
