diff --git a/setup.py b/setup.py.orig
index 49193f6..250e23a 100644
--- a/setup.py
+++ b/setup.py.orig
@@ -257,16 +257,16 @@ class PyBuildExt(build_ext):
         # The sysconfig variables built by makesetup that list the already
         # built modules and the disabled modules as configured by the Setup
         # files.
-        sysconf_built = sysconfig.get_config_var('MODBUILT_NAMES').split()
-        sysconf_dis = sysconfig.get_config_var('MODDISABLED_NAMES').split()
+        sysconf_built = [] # sysconfig.get_config_var('MODBUILT_NAMES').split()
+        sysconf_dis = [] # sysconfig.get_config_var('MODDISABLED_NAMES').split()
 
         mods_built = []
         mods_disabled = []
         for ext in self.extensions:
-            ext.sources = [ find_module_file(filename, moddirlist)
+            ext.sources = [ os.path.join('Modules', find_module_file(filename, moddirlist))
                             for filename in ext.sources ]
             if ext.depends is not None:
-                ext.depends = [find_module_file(filename, moddirlist)
+                ext.depends = [os.path.join('Modules', find_module_file(filename, moddirlist))
                                for filename in ext.depends]
             else:
                 ext.depends = []
@@ -521,7 +521,7 @@ class PyBuildExt(build_ext):
         # only change this for cross builds for 3.3, issues on Mageia
         if cross_compiling:
             self.add_gcc_paths()
-        self.add_multiarch_paths()
+        #self.add_multiarch_paths()

         # Add paths specified in the environment variables LDFLAGS and
         # CPPFLAGS for header and library files.
@@ -2389,8 +2389,8 @@ def main():
           # If you change the scripts installed here, you also need to
           # check the PyBuildScripts command above, and change the links
           # created by the bininstall target in Makefile.pre.in
-          scripts = ["Tools/scripts/pydoc3", "Tools/scripts/idle3",
-                     "Tools/scripts/2to3", "Tools/scripts/pyvenv"]
+          scripts = []#"Tools/scripts/pydoc3", "Tools/scripts/idle3",
+                      #"Tools/scripts/2to3", "Tools/scripts/pyvenv"]
         )
 
 # --install-platlib
