index 8d17d61..f51ac28 100644
--- a/Modules/_sqlite/module.h
+++ b/Modules/_sqlite/module.h
@@ -25,6 +25,10 @@
 #define PYSQLITE_MODULE_H
 #include "Python.h"

+#ifndef MODULE_NAME
+#define MODULE_NAME "_sqlite3"
+#endif
+
 #define PYSQLITE_VERSION "2.6.0"

 extern PyObject* pysqlite_Error;
--- a/Modules/_sqlite/sqlitecompat.h
+++ b/Modules/_sqlite/sqlitecompat.h
@@ -26,6 +26,10 @@
 #ifndef PYSQLITE_COMPAT_H
 #define PYSQLITE_COMPAT_H

+#ifndef MODULE_NAME
+#define MODULE_NAME "_sqlite3"
+#endif
+
 /* define Py_ssize_t for pre-2.5 versions of Python */

 #if PY_VERSION_HEX < 0x02050000