diff --git a/setup.py b/setup.py-patched
index 21e20ab..8b75108 100755
--- a/setup.py
+++ b/setup.py-patched
@@ -77,8 +77,8 @@ from _vendor.packaging.version import Version
 
 # Get geos_version from GEOS dynamic library, which depends on
 # GEOS_LIBRARY_PATH and/or GEOS_CONFIG environment variables
-from shapely._buildcfg import geos_version_string, geos_version, \
-        geos_config, get_geos_config
+#from shapely._buildcfg import geos_version_string, geos_version, \
+        #geos_config, get_geos_config
 
 logging.basicConfig()
 log = logging.getLogger(__file__)
