diff --git a/configure b/configure-patched
index 4c0435e..b067543 100755
--- a/configure
+++ b/configure-patched
@@ -5465,7 +5465,6 @@ $as_echo "#define Py_ENABLE_SHARED 1" >>confdefs.h
 	  LDLIBRARY='libpython$(VERSION).so'
 	  BLDLIBRARY='-Wl,-R,$(LIBDIR) -L. -lpython$(VERSION)'
 	  RUNSHARED=LD_LIBRARY_PATH=`pwd`${LD_LIBRARY_PATH:+:${LD_LIBRARY_PATH}}
-	  INSTSONAME="$LDLIBRARY".$SOVERSION
           ;;
     Linux*|GNU*|NetBSD*|FreeBSD*|DragonFly*|OpenBSD*)
 	  LDLIBRARY='libpython$(VERSION).so'
@@ -5476,7 +5475,6 @@ $as_echo "#define Py_ENABLE_SHARED 1" >>confdefs.h
 		SOVERSION=`echo $SOVERSION|cut -d "." -f 1`
 		;;
 	  esac
-	  INSTSONAME="$LDLIBRARY".$SOVERSION
 	  ;;
     hp*|HP*)
 	  case `uname -m` in
